grunt-configs/concat.js

```diff
@@ -0,0 +1,14 @@
+module.exports = function (grunt, options) {
+    return {
+        curlPlusCorePlusBoot: {
+            // https://github.com/gruntjs/grunt-contrib-concat/issues/131
+            // options: { sourceMap: true },
```

Comment:
Will remove this comment but I tried to get grunt concat to output source maps rebased on the input source maps, but it  won't work due to a bug :cry:. Thus, we can't have a source map for this file.